frontend/src/components/atoms/SnackBarContainer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+import SnackBarCloseButton from './SnackBarCloseButton';
+import { theme } from '../../styling/theme';
+import SnackBarLoader from './SnackBarLoader';
+
+interface ISnackBarProps {
+  content: string;
+  good: boolean;
+  className?: string;
+  speed?: string;
+  clicker?: () => void;
+}
+
+const LoaderSpeed = (speed = '6s') => {
+  if (speed === 'fast') {
+    return '4s';
+  } else if (speed === 'slow') {
+    return '8s';
+  } else {
+    return '6s';
+  }
+};
+
+const SnackBarContainer: React.FC<ISnackBarProps> = ({
+  content,
+  good,
+  className,
+  speed,
+  clicker,
+}) => {
+  return (
+    <div className={className}>
+      <p>{content}</p>
+      <SnackBarCloseButton onClick={clicker}>x</SnackBarCloseButton>
+      <SnackBarLoader
+        style={{
+          animationDuration: LoaderSpeed(speed),
+        }}
+      />
+    </div>
+  );
+};
+
+export default styled(SnackBarContainer)`
+  color: ${theme.palette.primary.contrast};
+  background-color: ${theme.palette.background.default};
+  padding: 0.5em;
+  border-radius: 3px;
+  border: 3px solid;
+  border-color: ${props =>
+    props.good ? theme.palette.success.main : theme.palette.danger.main};
```

Comment:
When updating all other occurrences of the `theme` -> `props.theme` thing, remember to update these as well.